application/view/common/block-layout/browse-preview.phtml

```diff
@@ -16,9 +16,15 @@ foreach ($this->resources as $resource):
     $body = $bodyTerm ? $resource->value($bodyTerm) : $resource->displayDescription();
 ?>
     <li class="<?php echo $this->resourceType; ?> resource">
-        <?php echo $resource->linkRaw($this->thumbnail($resource, 'medium', ['title' => $heading])); ?>
-        <h4><?php echo $resource->link($heading); ?></h4>
-        <?php if ($body): ?>
+        <?php
```

Comment:
My only substantive note: it's not ideal to be checking for the presence of the components in the array here, since it's inside the for loop, so we'll be re-searching over the array for every displayed resource.

These should probably be just checked once at the top and assigned to variables that are actually used in the `if` statements here.